pkgs/stdenv/generic/setup.sh

```diff
@@ -69,14 +121,14 @@ _logHook() {
     shift 2
 
     if declare -F "$hookExpr" > /dev/null 2>&1; then
-        nixLog "calling '$hookKind' function hook '$hookExpr'" "$@"
+        nixInfoLog "calling '$hookKind' function hook '$hookExpr'" "$@"
```

Comment:
I think these logs should use `nixTalkativeLog` (i.e. log level 4, see https://github.com/NixOS/nixpkgs/pull/331383#pullrequestreview-2211856772).

> Nix seems to filter out messages with level ≥ 4 by default.

Just in case we update `nixLog` functions to use structured logs.

These logs are informational but we have to temporarily disable them because they are a bit too _talkative_ for some use cases.

Reply:
Will do. I missed that comment.